optimum_benchmark/backends/pytorch.py

```diff
@@ -72,10 +73,88 @@ class PyTorchConfig(BackendConfig):
     disable_grad: bool = "${is_inference:${benchmark.name}}"  # type: ignore
     eval_mode: bool = "${is_inference:${benchmark.name}}"  # type: ignore
 
+    # training options
+    use_ddp: bool = False
+    ddp_config: Optional[Dict[str, Any]] = None
+
+    def __post_init__(self):
```

Comment:
really clean!

Does this non-greedy approach means that if defaults of optimum-benchmark are changed, a previous benchmark config yml may not be accurately loaded?

Reply:
it means that they'll rely on the defaults of `optimum-benchmark`. this won't be much of a problem for things like experimenting with the speedup of an optimization once, but it is a problem for regression tests, in which case it's better to keep using the same version of the repo, unless interested by a new feature, in which case the user will need to check everything is still the same. I guess we can add a warning for future changes whenever we're planning one, like `disable_xxx_fusion` in `optimum` optimizations.